src/nifgen/metadata/functions.py

```diff
@@ -18,33 +18,6 @@
         'python_name': 'abort',
         'returns': 'ViStatus'
     },
-    'AdjustSampleClockRelativeDelay': {
```

Comment:
Are changes in this file supposed to be part of this PR?

Reply:
It required #1086 first. I have changed the title to reflect that.

So until that one is accepted, merged into master, and then merged into this branch, the changes from there will show up here as well.